pkg/master/ports/ports.go

```diff
@@ -41,5 +45,5 @@ const (
 	KubeControllerManagerPort = 10257
 	// CloudControllerManagerPort is the default port for the cloud controller manager server.
 	// This value may be overridden by a flag at startup.
-	CloudControllerManagerPort = 10258
+	CloudControllerManagerPort = cloudprovider.CloudControllerManagerPort
```

Comment:
Is it worth while to pull in the k8s.io/cloud-provider dependency here if this constant is not likely to change anytime in the future?

Reply:
I did it due to the above discussion: https://github.com/kubernetes/kubernetes/pull/90734#discussion_r420958493 Just in case people would look for the port